src/pages/index.js

```diff
@@ -53,21 +34,94 @@ const cardModal = document.querySelector("#add-card-modal");
 const cardForm = cardModal.querySelector(".modal__form");
 const cardModalCloseBtn = cardModal.querySelector(".modal__close-btn");
 const cardModalBtn = document.querySelector(".profile__post-btn");
+const avatarModalBtn = document.querySelector(".profile__avatar-btn");
 const cardNameInput = cardModal.querySelector("#add-card-name-input");
 const cardLinkInput = cardModal.querySelector("#add-card-link-input");
-const cardSubmitBtn = cardModal.querySelector("#button");
+const submitBtn = document.querySelector("#Button");
 const previewModal = document.querySelector("#preview-modal");
 const previewModalImageEl = previewModal.querySelector(".modal__image");
 const previewModalText = previewModal.querySelector(".modal__caption");
 const previewModalCloseButton = previewModal.querySelector(
   ".modal__close-btn_type_preview"
 );
+//Avatar Form Modal
+const avatarModal = document.querySelector("#avatar-modal");
+const avatarForm = avatarModal.querySelector(".modal__form");
+const avatarModalCloseBtn = avatarModal.querySelector(".modal__close-btn");
+const avatarSubmitBtn = avatarModal.querySelector(".modal__submit-btn");
+const avatarInput = avatarModal.querySelector("#profile-avatar-input");
 
-const deleteButton = document.querySelector(".card__delete-button");
+//Delete Form elements
+
+const deleteModal = document.querySelector("#delete-modal");
+const deleteModalCloseBtn = document.querySelector("#delete-close");
+const deleteModalCancelBtn = document.querySelector("#cancel-button");
+const deleteForm = deleteModal.querySelector(".modal__delete-form");
 
 const cardTemplate = document.querySelector("#card-template");
 const cardsList = document.querySelector(".cards__list");
 
+let selectedCard;
+let selectedCardId;
+
+api
+  .getAppInfo()
+
+  .then(([cards, userInfo]) => {
+    console.log(cards); // Check if `cards` contains data
+    console.log(userInfo);
+    cards.forEach((item) => {
+      const cardElement = getCardElement(item);
+      console.log("Card element:", cardElement); // Verify it returns an element
+      cardsList.append(cardElement);
+    });
+    const userNameElement = document.querySelector(".profile__name");
+    const userDescriptionElement = document.querySelector(
+      ".profile__description"
```

Comment:
You've already found `profileName` and `profileDescription` in lines 21 and 22

It’s better to find **constant DOM elements** only 1 time at the top of the file so as not to waste resources on searching them again and again when you call a method (function) because searching is a very **hard** operation for the browser engine